api/src/main/java/org/openmrs/module/ModuleUtil.java

```diff
@@ -571,7 +576,15 @@ public static void expandJar(File fileToExpand, File tmpModuleDir, String name,
 					String entryName = jarEntry.getName();
 					// trim out the name path from the name of the new file
 					if (!keepFullPath && name != null) {
-						entryName = entryName.replaceFirst(name, "");
+						if (name.equals(entryName)) {
+							int lastSlash = name.lastIndexOf('/');
```

Comment:
Since `JarEntry` uses slash as a separator I would expect that `name` parameter will also use them. This is in line with rest of the code in this function, which also uses hard coded slash.